project.clj

```diff
@@ -1,8 +1,7 @@
-(defproject google-civic "2.0.2-SNAPSHOT"
+(defproject google-civic "2.1.0-SNAPSHOT"
   :description "Google Civic Info API client"
   :dependencies [[org.clojure/clojure "1.7.0"]
-                 [clj-http "2.0.0"]
-                 [cheshire "5.5.0"]
-                 [slingshot "0.12.2"]]
+                 [clj-http "2.3.0"]
```

Comment:
Sure, seems fine to stick w/ 2.x.x for that reason. Just wanted to note that it was more of a known quantity in terms of basic usage.